Framework/DataHandling/inc/MantidDataHandling/ReadMaterial.h

```diff
@@ -43,6 +44,10 @@ class DLLExport ReadMaterial {
     double unitCellVolume = EMPTY_DBL();
     /// The sample mass density to set, defaults to EMPTY_DBL()
     double sampleMassDensity = EMPTY_DBL();
+    /// The sample mass to set, defaults to EMPTY_DBL()
+    double sampleMass = EMPTY_DBL();
+    /// The sample volume to set, defaults to EMPTY_DBL()
+    double sampleVolume = EMPTY_DBL();
```

Comment:
In other places we've started using `boost::optional<double>` instead of `EMPTY_DBL`.

Given the number of times `isEmpty` is called around here, it might be a good place to refactor in a separate PR.